p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -376,4 +376,13 @@ Layout Description
         <readOnly/>
         <ecc/>
     </section>
+    <section>
+        <description>Ultravisor binary image (1MB)</description>
+        <eyeCatch>UVISOR</eyeCatch>
+        <physicalOffset>0x2F8E000</physicalOffset>
+        <physicalRegionSize>0x100000</physicalRegionSize>
+        <side>sideless</side>
+        <sha512Version/>
+        <ecc/>
```

Comment:
Trying to remember the meeting now .. did we say ECC was not needed?  It's not used for OPAL skiboot.

Is this supposed to be 1MB w/ ECC, or 1 MB logically speaking?  Probably we're ok either way,

Reply:
I can't quite recall either.. I think I remember people saying that 1MB is more than enough to fit the UV binary, so I though to throw ECC on top too.